docs/src/using-deis/registering-a-user.md

```diff
@@ -72,4 +65,4 @@ If you already have an account, use `deis login` to authenticate against the Dei
     Deis session information is stored in your user's ~/.deis directory.
 
 
-[controller]: ../understanding-deis/components.md#controller
```

Comment:
this link was accidentally removed. We have a link for it [here](https://github.com/deis/workflow/pull/163/files#diff-a2831b297cb4c9d8ab12c22c6d0e3665R3)